### PR TITLE
Rename steps in an evaluation

### LINKED ISSUE #267: Change progression names
Vi er interessert i å bytte navnet til etaper i BMT verktøyet i Fusion:
2. Fra "Preparation" til "Individual assessment" 
3. Fra "Alignment" til "Preparation"
4. Fra "Workshop" til "Workshop assessment"

### PR DESCRIPTION
**Pull request by ReedOnly**
- Hard code backend url for compatibility
- Rename Preparation -> Individual Assessment
- Rename Alignment -> Preparation
- rename Workshop -> Workshop Aassessment

closes #267

### CODE REVIEW

**Comment by ReedOnly** [comment]
I did not rename Workshop" to "WorkshopAssessment" in all the files and variables, maybe we should?

**Review by nicholasdalhaug — COMMENTED** [review]
Could we just use `Individual` instead of `IndividualAssessment`? I think we should ask Benjamin about some of this stuff. Also, this PR should not be merged before the database is updated / nuked.

File: frontend/src/components/AnswerSummarySidebar.tsx [hunk 1]
```diff
@@ -18,8 +18,8 @@ interface AnswerSummarySidebarProps
 const AnswerSummarySidebar = ({ open, onCloseClick, question, questionNumber, viewProgression }: AnswerSummarySidebarProps) => {
     const answers = question.answers.filter(answer => progressionLessThan(answer.progression, viewProgression))
 
+    const IndividualAnswers = answers.filter(a => a.progression === Progression.Individual)
```
Comment (ReedOnly) [inline frontend/src/components/AnswerSummarySidebar.tsx:21]: Should this start with capital letter or not?

Comment (nicholasdalhaug) [reply frontend/src/components/AnswerSummarySidebar.tsx:21]: I think variable names should start with a lower-case letter.

File: backend/api/Models/Models.cs [hunk 2]
```diff
@@ -32,6 +32,8 @@ public class Evaluation
         [Required]
         public Progression Progression { get; set; }
         [Required]
+        public ActiveStatus ActiveStatus { get; set; }
```
Comment (Christdej) [inline backend/api/Models/Models.cs:35]: We already have a enum called status, use that

File: backend/api/Startup.cs [hunk 3]
```diff
@@ -193,7 +193,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILogger<
             });
 
             // Comment out to use locally without authentication
-            app.UseAuthorization();
+            // app.UseAuthorization();
```
Comment (Christdej) [inline backend/api/Startup.cs:196]: Uncomment this

**Review by Christdej — CHANGES_REQUESTED** [review]
Good good, remember to update the DB and Codacy had some notes I would consider fixing

File: backend/api/Models/Models.cs (hunk 2 above)
Comment (nicholasdalhaug) [reply backend/api/Models/Models.cs:35]: This was actually my suggestion. I think this is a renaming of the old `Status`. I thought that `Status` was actually not very descriptive.

Comment (nicholasdalhaug) [reply backend/api/Models/Models.cs:35]: But shouldn't that renaming also be in this file? I cannot see it.

File: frontend/src/views/Evaluation/EvaluationView.tsx [hunk 4]
```diff
@@ -34,33 +34,33 @@ const EvaluationView = ({evaluation, onProgressEvaluationClick, onProgressPartic
                 />
             </Step>
             <Step
-                title="Preparation"
-                description={calcProgressionStatus(evaluation.progression, Progression.Preparation)}
-                stepKey={Progression.Preparation}
+                title="Individual Assessment"
```
Comment (nicholasdalhaug) [inline frontend/src/views/Evaluation/EvaluationView.tsx:37]: Here and on the lines below we can use the function `progressionToString(...)`.

File: backend/api/Models/Models.cs (hunk 2 above)
Comment (Christdej) [reply backend/api/Models/Models.cs:35]: What about a boolean value called Active?

File: frontend/src/views/Evaluation/Preparation/PreparationView.tsx [hunk 5]
```diff
@@ -47,61 +55,85 @@ const PreparationView = ({evaluation, onNextStepClick, onProgressParticipant}: P
...
+                                    viewProgression={viewProgression}
+                                />
+                            </div>
+                        )
+                    })}
+                </Box>
+                <Box>
+                    { selectedQuestion && selectedQuestionNumber &&
+                        <AnswerSummarySidebar
+                            open={selectedQuestion != undefined}
```
Comment (nicholasdalhaug) [inline frontend/src/views/Evaluation/Preparation/PreparationView.tsx:126]:
```suggestion
                            open={selectedQuestion !== undefined}
```

**Review by nicholasdalhaug — CHANGES_REQUESTED** [review]

File: frontend/src/views/Evaluation/Preparation/PreparationView.tsx (hunk 5 above)
Comment (ReedOnly) [reply frontend/src/views/Evaluation/Preparation/PreparationView.tsx]: fixed!

File: frontend/src/views/Evaluation/EvaluationView.tsx (hunk 4 above)
Comment (ReedOnly) [reply frontend/src/views/Evaluation/EvaluationView.tsx]: fixed!

**Review by Christdej — APPROVED** [review]

**Review by nicholasdalhaug — APPROVED** [review]

**Pull request merged**